dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/executor/workflow/StopWorkflowInstanceExecutorDelegate.java

```diff
@@ -95,6 +95,7 @@ void stopInMaster(WorkflowInstance workflowInstance) {
         } catch (ServiceException e) {
             throw e;
         } catch (Exception e) {
+            log.error("WorkflowInstance stop failed, workflowInstanceId: {}, workflowInstanceName: {}", workflowInstance.getId(), workflowInstance.getName(), e);
```

Comment:
Can we remove this log? Since the exception stack will be logged by aspect.

Reply:
the following ServiceException didn't contain the original call stack, it won't print to api log, so i added this.

Reply:
it's not a good way, i'll remove it first.

Reply:
Ok, I got you, since ServiceException are seems as expected exception, although it contains the origin stack, but will not log, so we need to change the aspect code to log the exception stack.